apps/sms/test/marionette/lib/views/conversation/accessors.js

```diff
@@ -0,0 +1,49 @@
+var SELECTORS = Object.freeze({
+  main: '#thread-messages',
+  message: '.message .bubble',
+  headerTitle: '#messages-header-text',
+  container: '#messages-container',
+  callButton: '#messages-call-number-button'
+});
+
+module.exports = {
+  get message() {
+    return client.helper.waitForElement(SELECTORS.message);
+  },
+
+  get headerTitle() {
+    return client.helper.waitForElement(
+      SELECTORS.headerTitle
+    );
+  },
+
+  getMessageContent: function(message) {
+    return client.helper.waitForElement(
+      message.findElement(SELECTORS.content)
+    );
+  },
+
+  findMessage: function(id) {
+    return client.findElement('.message[data-message-id="' + id + '"]');
+  },
+
+  scrollUp: function() {
+    var conversationContainer = client.findElement(
+      SELECTORS.container
+    );
+
+    actions.flick(conversationContainer, 50, 50, 50, 350).perform();
+  },
+
+  get callButton() {
+    return client.helper.waitForElement(SELECTORS.callButton);
+  },
+
+  waitToAppear: function() {
```

Comment:
This method performs a wait action on a single an HTML element. I wanted to split it into 2 (`waitToBeInTheDom` and `waitToBeAtTheRightPosition`) but `waitToBeAtTheRightPosition` had mostly the same code than this current function. I prefered to merge them then.

Also the method returns the HTML element. I'd say, it belongs to the accessors.